ci/buildspec_main_backend.yml

```diff
@@ -0,0 +1,37 @@
+# AWS CodeBuild config
+# This file is referenced in the infra repo as a CodeBuild buildspec target.
+
+# These 5 variables are set in the codebuild project definition's environment secion.
+# ECR_REGION
+# ECR_ACCOUNT_ID
+# ECR_NAMESPACE
+# CLOUDINARY_NAME
+# FILE_SERVICE
```

Comment:
Do we want to have these align with the env variables that already exist?  For example, is `CLOUDINARY_CLOUD_NAME` in Public & Partners the same as `CLOUDINARY_NAME` here?  And I assume ECR Region, Account ID and Namespace are different than the AWS vars I have in my files, but if not, should those be the same? `FILE_SERVICE` is consistent with what I have in my env file.

Reply:
_done_